torch/tensor.py

```diff
@@ -381,6 +381,11 @@ def __ixor__(self, other):
             raise RuntimeError('logical operations are supported on ByteTensors only')
         return self.copy_((self + other).eq(1))
 
+    def __invert__(self):
+        if (type(self).__name__ != 'ByteTensor'):
```

Comment:
Please remove these parenthesis, they're not needed (they are in the lines above because we needed to break lines).